runtime/parachains/src/session_info.rs

```diff
@@ -196,10 +210,20 @@ impl<T: Config> Pallet<T> {
 			dispute_period,
 		};
 		Sessions::<T>::insert(&new_session_index, &new_session_info);
-		SessionExecutorParams::<T>::insert(
-			&new_session_index,
-			ExecutorParams::from(&EXECUTOR_PARAMS[..]),
-		);
+
+		let new_executor_params = if let Some(executor_params) = <PendingExecutorParams<T>>::get() {
```

Comment:
Should I add a migration to put `None` to the `PendingExecutorParams` for the first time?

Reply:
No